requirements/dev.txt

```diff
@@ -4,33 +4,39 @@
 #
 #    make upgrade
 #
--e file:///home/cpennington/work/code/edx-lint  # via -r requirements/dev.in
```

Comment:
This is the line that prompted this PR cc @cpennington

Reply:
Should we assume this was just a mistake and remove it?

Reply:
The problem is that if we add `-e .` to the `dev.in` file, it will automatically re-compile to an absolute path in the `dev.txt` file because of https://github.com/jazzband/pip-tools/issues/204.
Note that the cookiecutter does not add `-e .` to dev.in (for this good reason): https://github.com/edx/edx-cookiecutters/blob/master/cookiecutter-django-app/%7B%7Bcookiecutter.repo_name%7D%7D/requirements/dev.in

Reply:
I am assuming that merging both `-e .` and `-e file:///home/cpennington/work/code/edx-lint` were just mistakes.  Right?

Reply:
I guess that adding `-e .` was intended, but the compilation to an absolute path wasn't.

Reply:
Oops. Yeah, definitely didn't mean to add an absolute path here.